js/formidable_admin.js

```diff
@@ -10926,6 +10926,21 @@ function frmAdminBuildJS() {
 
 			// Set fieldsUpdated to 0 to avoid the unsaved changes pop up.
 			frmDom.util.documentOn( 'submit', '.frm_settings_form', () => fieldsUpdated = 0 );
+
+			const manageStyleSettings = document.getElementById( 'manage_styles_settings' );
+			if ( manageStyleSettings ) {
+				manageStyleSettings.addEventListener(
+					'change',
+					event => {
+						const target = event.originalTarget;
```

Comment:
```suggestion
						const target = event.target;
```
Using target is better in our case and I tested it and it's working properly.
https://developer.mozilla.org/en-US/docs/Web/API/Event/originalTarget

What do u think?

Reply:
@engahmeds3ed Thanks. I don't have any reason to use originalTarget in particular. I just updated this.